crates/compiler/test_mono/generated/inspect_derived_record.txt

```diff
@@ -1,18 +1,18 @@
 procedure #Derived.0 (#Derived.1):
-    let #Derived_gen.0 : {Decimal, I64} = CallByName Inspect.30 #Derived.1;
+    let #Derived_gen.0 : {I64, Decimal} = CallByName Inspect.30 #Derived.1;
```

Comment:
but, this will just fail on an AARCH64 system right? there decimal still has a higher alignment

Reply:
Ah yeah, I guess those tests aren't safe cross architecture unless we pin the alignment of types across all architectures.